### PR TITLE
Display pending withdrawals during unstaking

### PR DESCRIPTION
**Pull request by p-m-jakubowski**
- Adds `PendingWithdrawal` component
- Displays list of pending withdrawals below the unstaking panel

### Screenshots

Screenshot 2022-07-25 at 15 48 04

#### Withdrawals that are being claimed

Screenshot 2022-07-25 at 15 50 43

### CODE REVIEW

File: src/features/unstake/Unstake.tsx [hunk 1]
```diff
@@ -27,6 +28,12 @@ export const Unstake = () => {
         fromToken="stCELO"
         toToken="CELO"
       />
+      {pendingWithdrawals.length > 0 ? (
```
Comment (krystian50) [inline src/features/unstake/Unstake.tsx:31]: Could it be && operator? Cant remember if react renders boolean value.

File: tailwind.config.js [hunk 2]
```diff
@@ -4,39 +4,40 @@ const colors = require('tailwindcss/colors');
...
-      red: '#F17256',
-      orange: {
-        DEFAULT: '#E7B65D',
-        700: '#E7B65D',
-        800: '#DFA849',
-        900: '#EEAF43',
+    extend: {
+      colors: {
+        // default
+        black: colors.black,
```
Comment (krystian50) [inline tailwind.config.js:10]: Extend doesn’t mean we do not have to add default values?

**Review by krystian50 — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 2 above)
Comment (p-m-jakubowski) [reply tailwind.config.js]: Right. Removed.

File: src/features/unstake/Unstake.tsx (hunk 1 above)
Comment (p-m-jakubowski) [reply src/features/unstake/Unstake.tsx]: Treats `false` the same as `null` :+1:

**Review by krystian50 — APPROVED** [review]

**Pull request merged**